app/src/main/res/values/styles.xml

```diff
@@ -15,12 +15,28 @@
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
     </style>
-    
+
+    <style name="AppTheme.NoActionBar.TranslucentStatus">
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
+        <item name="android:windowTranslucentStatus">true</item>
+        <!--Android 5.x should set statusBarColor with transparent, or It will be grey-->
+        <item name="android:statusBarColor">@android:color/transparent</item>
+    </style>
+
     <style name="AppTheme.SplashScreen" parent="Theme.SplashScreen">
```

Comment:
Why create a new style with the suffix `.TranslucentStatus` when the settings could be integrated into the existing AppTheme styles?

Reply:
Perhaps, the existing theme will be referred to later? so, I made a copy.

Reply:
The new style has been merged into the existing AppTheme in the latest commits.